ci-project/models/overview.md

```diff
@@ -9,3 +9,12 @@ This [dbt](https://www.getdbt.com/) project is for demonstrations and tutorials.
 The source code can be found [here](https://github.com/clrcrl/jaffle_shop).
 
 {% enddocs %}
+
+{% docs __jaffle_shop__ %}
+## Jaffle_shop Project level overview example
+This [dbt](https://www.getdbt.com/) project may have only one project (`jaffle_shop`) currently.
+However there may be other projects included in your own projects.
+
+You can assign a unique overview for each project by adding a docs block in an .md file of your project
+and giving it a name with the following convention __<project_name>__.
+{% enddocs %}
```

Comment:
* Could you add an entry for `{% docs __dbt_utils__ %}` below?
* As Drew mentioned, to really see your change in the PR preview, we'll need to run `dbt docs generate` in the `ci-project` to replace the artifacts currently in `data/`. If you're up for doing this as part of this PR, great! If not, we can do it separately after merging.

Reply:
@jtcohen6  Thanks for the suggestion I added a block for dbt_utils. I also generated docs in the ci-project and replaced the data/ files with them. I also signed the CLA. Please let me know if I did anything incorrectly with the data/ artifacts.